types/resolve/index.d.ts

```diff
@@ -87,6 +87,8 @@ declare function resolveSync(id: string, opts?: resolve.SyncOpts): string;
  */
 declare function resolveIsCore(id: string): boolean | undefined;
 
+type PackageJson = Record<string, unknown>;
```

Comment:
seems even better if this enforces proper JSON types:
```suggestion
// https://github.com/Microsoft/TypeScript/issues/3496#issuecomment-128553540
type JSONValue = string | number | boolean | JSONObject | JSONArray;
interface JSONObject {
    [x: string]: JSONValue;
}
interface JSONArray extends Array<JSONValue> { }

type PackageJSON = JSONObject;
```
(note the proper initialism capitalization also)